install/receivers.csv

```diff
@@ -39,9 +39,9 @@ Thales,ASHTECH Z-XII3,LP419994102,VAVS,2000-09-20T00:00:00Z,2000-09-20T23:59:59Z
 Thales,ASHTECH Z-XII3,LP419994102,AUCK,2001-10-28T02:17:00Z,2002-12-01T00:01:00Z
 Thales,ASHTECH Z-XII3,LP419994102,AUCK,2003-01-09T02:00:00Z,2005-11-03T03:59:00Z
 Thales,ASHTECH Z-XII3,LP419994103,MQZG,2001-09-03T00:00:01Z,2005-02-28T03:00:00Z
-Trimble Navigation Ltd.,TRIMBLE 4000SSE,7590,OUSD,1995-01-16T00:00:00Z,1999-08-09T00:00:00Z
-Trimble Navigation Ltd.,TRIMBLE 4000SSE,7590,OUSD,1999-08-13T00:00:00Z,2005-11-17T00:00:00Z
-Trimble Navigation Ltd.,TRIMBLE 4000SSI,18169,OUSD,2005-11-17T00:00:00Z,2006-01-31T00:00:00Z
+Trimble Navigation Ltd.,TRIMBLE 4000SSE,7590,OUSD,1995-01-16T00:00:00Z,1999-08-08T23:59:59Z
+Trimble Navigation Ltd.,TRIMBLE 4000SSE,7590,OUSD,1999-08-13T00:00:00Z,2005-11-16T23:59:59Z
```

Comment:
I think the receiver name for this one may be wrong, as before 2005-11-16 is TRIMBLE 4000SSI not 4000SSE.